### ISSUE TITLE
Ad serving is limited.

### ISSUE THREAD

**Issue by arunchaudharyweb** [googleads/googleads-mobile-flutter#268]
Is anyone else experiencing the same issue?

I have followed all the best practices for native advanced and rewarded ads. But the ad serving limit won't disappear whatever I try. 

See the screenshot of my implementation for native ads.

My all downloads are organic. I don't see any other issue with my application.

I think there is some problem with this package.

Here is my application link on the play store: https://play.google.com/store/apps/details?id=com.fitnessband.miband6 if anyone wants to see ads implementation.
<img src="https://user-images.githubusercontent.com/59797321/121107362-6e9c1500-c825-11eb-9e2a-93309ae7f463.jpg" height="450" />

**Comment by vishalknishad** [comment]
Wait for some days, the warning might go. If not, then delete all the ads from admob and wait for 2 weeks.
Unfortunately, this is only what we can do.

**Comment by arunchaudharyweb** [comment]
It has been more than 2 months since the ad serving limit. I can't delete the ad units because I have a subscription in my app that allows users to upgrade for an ad-free experience. Otherwise, the subscription will be meaningless.
Are there any other options?
Is there any software engineer from Google that can tell what's going on?  
Please have a look at my application.

**Comment by arunchaudharyweb** [comment]
I closed my current Admob account and made a new one. It's working fine now.

**Comment by vishalknishad** [comment]
Is it still working for you. I got ad limit on another account also.

**Comment by arunchaudharyweb** [comment]
Google AdMob sucks. For no reason, it gives an ad serving limit. Try FAN. It's better than AdMob. I wasted my 6 months learning and building android applications. I am also paying the server and storage fees from my pocket. 

For the last 6 months, I haven't received a single penny from admob. I closed my last AdMob account where I earned 60$. They haven't paid the money yet. Then I opened a new AdMob account. I have earned 60$ in this account as well but I got an ad serving limit again. It's been around for 32 days now.

I suggest learning to build decentralized applications where these companies have no control. You will definitely benefit there if you put in some work. I have learnt a lot in the last few months. Soon, I also going to launch my own token on the Ethereum blockchain.

**Comment by TahaTesser** [comment]
Hi @arunchaudharyweb 
It looks like a common issue even in native app development, I am closing duplicate of https://github.com/googleads/googleads-mobile-flutter/issues/268

**Comment by liondev7** [comment]
hi,

TLDR : remove native ad units, there is something wrong with them in the implementation.

Details:

i had exactly the same problem, after doing many tests i think i found the issue, it's the native ads, there is definitely something wrong with the native ads implementation with this package, that makes google think you're doing something wrong or trying to do ad fraud even if you do all best practices.

Here is what worked for me, remove all ad units from Admob and from your app, wait for a week or so for the ad serving limit to be lifted, then create your ad units back and put back ads into your app BUT DO NOT USE NATIVE ADS, i figured this out the hard way, when i put back the ads (interstitial + native) i get the ad serving limited, when i removed native and ONLY put back interstitials, it's working fine for me so far.

(NB : i'm implementing all best practices as well, and traffic is organic).

**Comment by elkhalifte** [comment]
Banner ads -> limit as well , because the user will still get ad impressions even if the app is in background.

Lifecycle events are required in this case to put banner in pause while application is in background.

I'll make a pull request for this when am free.

**Comment by liondev7** [comment]
update: 
now i'm getting account limited with interstitials only, no native ads no banners.
Again, traffic is 100% organic downloads, admob account with 1 single app, all best practices implemented to the smallest details, this is really unfortunate.

**Comment by elkhalifte** [comment]
@liondev7 what i can say is , you might be showing interstitial on app first start , or when user exists the app , or maybe you are showing multiple interstitials for users , for this try to show no more than 1 interstitial per minute per each user.

**Comment by liondev7** [comment]
Hmm, I’m not doing any of the things you mentioned, the app converts files, the interstitial is only displayed after file conversion succeeds, so only a small portion of users actually reaches that step and sees the interstitial, and i guess it happens only a couple of times per session, nothing weird or sketchy, best practices respected, yet account limitation hits the account as soon as it’s lifted.

With no way to appeal nor request human/manual review from google, nor contact support, this displays how much google disrespects the hardworking developers and it doesn’t care about them at all to be honest.

**Comment by elkhalifte** [comment]
Yeah that's weird, and i agree with you.

**Comment by GauravCalidig** [comment]
Facing same issue

**Comment by elkhalifte** [comment]
Flutter + Admob = limit

**Comment by CalvinTp** [comment]
I am facing this issue as well. I am using banner ads in my app.
